docs/Accessibility - Principles and Requirements.md

```diff
@@ -0,0 +1,138 @@
+
+## Motivation
+
+Accessibility is an important feature that ensures that all users,
+including those with disabilities, can use the Mozilla VPN Client
+effectively. Following accessibility principles and requirements
+enhances the user experience, reaches a broader audience, and complies with
+accessibility standards across all supported platforms.
+
+## Principles
+
+1. **Inclusive design:** The VPN Client should be designed for use by
+      all people, as far as possible, without the need for specialized
+      adaptations. By considering the diverse needs of users from the
+      start, we can create a more inclusive and higher quality user
+      experience.
+
+2. **Compatibility with major Assistive Technologies (ATs):** The VPN
+      Client should be compatible with the major ATs of the platforms
+      that it runs on, including Text Size, Screen Readers, Keyboard
+      Access, Text Contrast, Screen Magnification, and others.
+
+3. **Alignment with Established Standards:** The VPN Client should try
+      to meet accessibility standards like
+      [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
+      where applicable, even though the VPN Client is not a Web App.
+
+4. **Foster Collaborative Development and Testing:** Promote a
+      collaborative approach among developers, testers and accessibility
+      experts during the development and testing of Accessibility
+      features in the VPN client. Test with real users. This will result
+      in higher quality implementations, continuous improvement and
+      early detection of accessibility issues.
+
+## Accessibility Requirements for VPN Client
+
+For more information about these requirements, testing, and
+prioritization principles and guidelines, please refer to [Mozilla VPN
+Accessibility \-- Resources for
+Testing](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/docs/Accessibility%20-%20Testing%20Resources.md).
+
+### High Priority Requirements
+
+1. **Text Resizing:**
+
+     - Users should be able to adjust the text size through the platform\'s
+      settings while ensuring that the application text remains readable
+      and all functionality remains intact.
+
+2. **Screen Reader Compatibility:**
+
+     - All screens must be compatible with major screen readers, allowing
+      users with vision impairments to access the content and obtain
+      information about interactive elements.
+
+     - All screens must be navigable using Screen Reader keyboard
+      shortcuts.
+
+3. **Keyboard Access:**
+
+     - All functionality and interactive elements must be accessible solely
+      using a keyboard, without relying on mouse or touch interactions.
+
+4. **Text Contrast Compliance:**
+
+     - The default text contrast against its background must meet the
+      standards set by
+      [WCAG](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast).
+
+     - Users with vision impairments should be able to apply the
```

Comment:
Do we think `High Contrast Theme Support` should be it's own numbered item?

Reply:
Yes, will do.